### ISSUE TITLE
unified html?

### ISSUE THREAD

**Issue by fippo** [legastero/jingle-interop-demos#4]
After stealing the placeholders from @lloydwatkin I wondered whether we should attempt to use exactly the same html and outsource our JS. Thoughts?

**Comment by lloydwatkin** [comment]
Good idea. Also means we could all benefit from work each other does. 

 for one would like to see:
- incoming presence subscription data
- a disco#info on the presence friends
- click to call
- \+ standard c&p over the other person's jid

Makes it a much nicer experience.
